rooms.js

```diff
@@ -932,17 +932,25 @@ class BattleRoom extends Room {
 		this.resetUser = '';
 	}
 	requestKickInactive(user, force) {
-		if (this.resetTimer) {
-			if (user) this.sendUser(user, '|inactive|The inactivity timer is already counting down.');
+		if (this.resetTimer && user && (this.resetUser === user.userid || (user in this.game.players && this.resetUser === '+') || this.resetUser === '~')) {
```

Comment:
If `user` evaluates to false, won't the condition short-circuit before it gets to the stuff that will crash? What used to be line 989 (now 997) seems to have been relying on that before I changed it, and still is. `user` needs to be tested again because that if statement can evaluate to false even if `user` doesn't.